### PR TITLE
feature e2e users endpoint

### PR DESCRIPTION
**Pull request by celestepalet**
Hi, I added more scenarios e2e for users endpoint, please review my code! Thanks
Best Regards
Celeste

### CODE REVIEW

File: wordpress/resources/json/get_user_schema.json
Comment (Fabian2025) [inline wordpress/resources/json/get_user_schema.json:1]: please add new line in this file

**Review by Fabian2025 — COMMENTED** [review]
Hi Celeste I put a comment in a json schema

File: wordpress/resources/json/get_user_schema.json
Comment (celestepalet) [reply wordpress/resources/json/get_user_schema.json:1]: Added!

Comment (Fabian2025) [reply wordpress/resources/json/get_user_schema.json:1]: ok

**Review by Fabian2025 — APPROVED** [review]

**Pull request merged**